scipy/linalg/flapack_sym_herm.pyf.src

```diff
@@ -30,22 +31,43 @@ subroutine <prefix2>syev(compute_v,lower,n,w,a,work,lwork,info)
 
 end subroutine <prefix2>syev
 
-subroutine <prefix2c>heev(compute_v,lower,n,w,a,work,lwork,rwork,info)
```

Comment:
I've only done about two of these, so I'm not the most qualified person to review. But to do this properly would take a substantial fraction of the time it took to write this stuff. Also, the tests for the `_lwork` routines aren't very strong, so we can't really review by testing. Thoughts on how to ensure quality?

Reply:
I am hoping that it is tested through the eigh and eigvalsh tests for the others but yes it needs more tests.